osu.Game/OsuGame.cs

```diff
@@ -485,10 +485,19 @@ private void load()
             }
         });
 
-        public void OpenUrlExternally(string url, bool bypassExternalUrlWarning = false) => waitForReady(() => externalLinkOpener, _ =>
+        public void OpenUrlExternally(string url, bool forceBypassExternalUrlWarning = false) => waitForReady(() => externalLinkOpener, _ =>
         {
+            bool isTrustedDomain;
+
             if (url.StartsWith('/'))
-                url = $"{API.APIEndpointUrl}{url}";
+            {
+                url = $"{API.WebsiteRootUrl}{url}";
+                isTrustedDomain = true;
+            }
+            else
+            {
+                isTrustedDomain = url.StartsWith(API.APIEndpointUrl, StringComparison.Ordinal);
```

Comment:
What's with the mixed usage of `WebsiteRootUrl` and `APIEndpointUrl` here? I guess in practice it doesn't matter because we no longer have a configuration wherein the two would lead to different places anymore, but surely I'd expect the same one to be used in both places (likely `WebsiteRootUrl`)?

Reply:
Should be `WebsiteRootUrl`, will change it.